internal/cli/runner_agent.go

```diff
@@ -272,6 +276,14 @@ func (c *RunnerAgentCommand) Flags() *flag.Sets {
 			Name:   "odr",
 			Target: &c.flagODR,
 			Usage:  "Indicates to the runner it's operating as an on-demand runner.",
+			Hidden: true,
+		})
+
+		f.StringVar(&flag.StringVar{
+			Name:   "odr-profile-id",
+			Target: &c.flagOdrProfileId,
+			Usage:  "The ID of the odr profile used to create the task that is running this runner.",
+			Hidden: true,
```

Comment:
Should these lines all be indented or is GitHub just being weird about how they're displayed?

Reply:
Github weirdness - it's just one stop.